keyboards/vxe75/vxe75.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright (C) 2024 Westberry Technology Corp., Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const aw_led g_aw_leds[RGB_MATRIX_LED_COUNT] = {
+/* Refer to IS31 manual for these locations
+ *   driver
+ *   |  R location
+ *   |  |          G location
+ *   |  |          |          B location
+ *   |  |          |          | */
+    {0, CS1_SW1,  CS2_SW1,  CS3_SW1   },  // 0
```

Comment:
<https://docs.qmk.fm/#/feature_rgb_matrix?id=aw20216s>

Location names have been updated, for example
```
    { 0, SW1_CS1, SW1_CS2, SW1_CS3 },
    { 0, SW1_CS4, SW1_CS5, SW1_CS6 },
    { 0, SW1_CS7, SW1_CS8, SW1_CS9 },
```

These should be updated